libs/wallet-ui/src/components/chrome/service-status.tsx

```diff
@@ -12,9 +12,10 @@ export function ServiceStatus() {
     dispatch,
     state: { network, networkConfig, serviceStatus },
   } = useGlobal()
-  const serviceUrl = networkConfig
-    ? `http://${networkConfig.host}:${networkConfig.port}`
-    : ''
+  const serviceUrl =
+    service.TYPE === 'http' && networkConfig
+      ? `http://${networkConfig.host}:${networkConfig.port}`
```

Comment:
**thought (non-blocking)** : It might be enforced properly elsewhere but it is possible someone wouldn't provide a port (expecting the default to work) and this connection string would be invalid. But I think it's a small chance.

Reply:
totally valid concern. tried it out, and when i import a network config without a port, the wallet api defaults the value to `0`. this is not ideal, but the port property won't go undefined. we're planning to phase out the port and host from the network configuration, as it has nothing to do with a network, it's an internal preference to your wallet app